### ISSUE TITLE
Auto centering a dynamic fancybox

### ISSUE THREAD

**Issue by MikeChav** [fancyapps/fancybox#2061]
Hello, 
I tried browsing through the existing threads and could not find an answer.

I am loading some fanyboxes which retrieve dynamic content via ajax. I don't seem to have a problem with the fancybox size's dimensions adjusting to the content size, but I would like to know if there is a built in way, or another way (besides using JS to adjust the margin) to center the fancybox as soon as the content is loaded. 

In fancybox 2 I would use `$.fancybox.reposition()`
I tried `$.fancybox.getInstance().update()`, to no avail.
Thank you!

**Comment by fancyapps** [comment]
Hi,

Actually, v3 uses "inline-block" vertical align trick, therefore content automatically centers. If not, then, please, create a demo, I would like to see that.

btw, there is currently no better solution for vertical centering, even flexbox is not perfectly suited for this due to scrolling and padding issue (for example, see https://stackoverflow.com/questions/47635911/how-to-use-safe-center-with-flexbox and other posts)

**Comment by MikeChav** [comment]
Thanks. It seems someone else's code was conflicting with mine.
